src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -29,6 +29,15 @@
                                     :account                account
                                     :test-networks-enabled? test-networks-enabled?
                                     :token-symbol           (get-in data [:asset-to-pay :symbol])}))
+         received-asset         (if-not (nil? asset-to-receive)
```

Comment:
currently default assets for new account are "ETH", "DAI" and "SNT",  I added this part to fallback to "SNT" if there is no "USDC" in user's account, we can remove this part after we implement #22160